src/components/InvalidAssignment/InvalidAssignment.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./InvalidAssignment.scss";
+
+function InvalidAssignment(props) {
+  function handleClose() {
+    props.setMessage([]);
+  }
+
+  function displayMessage(message) {
+    return <div className="Message">{message}</div>;
+  }
+
+  return (
+    <div className="InvalidAssignment">
+      <div className="MessageArea">{props.message.map(displayMessage)}</div>
+      <div className="CloseMessage" onClick={() => handleClose()}>
```

Comment:
same thing as I said before about button `onclicks`- you don't need the `()=> functionname()`